File: GTTS/INL.BudgetsService/Models/ExportEstimatedBudgetCalculator_Params.cs
using System;

namespace INL.BudgetsService.Models
{
    public class ExportEstimatedBudgetCalculator_Params
    {
        public string TrainingEventName { get; set; }
        public DateTime TrainingEventStart { get; set; }
        public DateTime TrainingEventEnd { get; set; }
        public ExportEstimatedBudgetCalculatorModel EstimatedBudgetModel { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.BudgetsService.Models/export-estimated-budget-calculator_params.ts


import { ExportEstimatedBudgetCalculatorModel } from './export-estimated-budget-calculator-model';

export class ExportEstimatedBudgetCalculator_Params {
  
	public TrainingEventName: string = "";
	public TrainingEventStart: Date = new Date(0);
	public TrainingEventEnd: Date = new Date(0);
	public EstimatedBudgetModel?: ExportEstimatedBudgetCalculatorModel;
  
}


